cypress/integration/finance/funds/funds.search.spec.js

```diff
@@ -65,7 +65,7 @@ describe('ui-finance: Funds list search and filter', () => {
     cy.deleteFundApi(fund.id);
   });
 
-  it('C4059 should return funds according to fund filters', { tags: [testTypes.smoke] }, function () {
+  it('C4059 Test the search and filter options for funds', { tags: [testTypes.smoke] }, function () {
```

Comment:
`{ tags: [testTypes.smoke] }`

Why in other places you use 

`{ tags: [testType.smoke] }`

I mean we need to decide what we should use: testType. or testTypes.

Reply:
I'll change it for testType